CustomCommandExamples/README.md

```diff
@@ -88,6 +89,14 @@ See [Installing DataMiner Integration Studio](https://aka.dataminer.services/Dis
 
 2. The custom command will be executed and the JSON example should be displayed as the response. <details><summary>`show demo`</summary>![Gif-OutputJson](https://user-images.githubusercontent.com/33500507/192817316-fc4de34d-a34e-42cd-a5fa-e8e99b363c2d.gif)</details>
 
+#### Input Parameter
+
+1. In Teams, execute the command '*run Input Parameter Example*'.
+
+2. A prompt will request the input configured in the automation script. Fill in any value and click the '*Run*' button.
+
+3. The custom command will be executed and the provided input value should be displayed as the response. <details><summary>`show demo`</summary>![Gif-InputParameter](https://user-images.githubusercontent.com/33500507/233356486-bc64d385-997e-4b1b-8466-e26c801a7e4d.gif)</details>
```

Comment:
If something is the result of a user action instead of an actual action they need to take, it should not be added as a numbered step. Instead, this is usually added as an indented line underneath the step it is a result of:

```suggestion
   The custom command will be executed, and the provided input value should be displayed as the response. <details><summary>`show demo`</summary>![Gif-InputParameter](https://user-images.githubusercontent.com/33500507/233356486-bc64d385-997e-4b1b-8466-e26c801a7e4d.gif)</details>
```